woocommerce/woocommerce-ios
# Quick site select

Closes #661. 

This PR sets up the quick site select feature, by navigating to Settings and tapping on the switch store row.

http://cld.wthms.co/arJC4W

## To test
1. While already logged in, go to My store > Settings. Check that the section title says "Selected Store". 
2. Tap the Switch Store row. Check that the currently selected store has a checkmark beside it.
3. Choose a different store. Load the other tabs, Orders List and Notifications, to verify the correct data is loading.
4. Log out by visiting My store > Settings Log Out Account.
5. Log in again. Note that the site picker defaults to selecting the first site.
(Try anything else to break it!)

@bummytime @astralbodies ready for review!

Update release notes:
